tests/topotests/multicast-pim-static-rp-topo1/test_multicast_pim_static_rp.py

```diff
@@ -0,0 +1,4204 @@
+#!/usr/bin/env python
+
+#
+# Author: Sarita Patra <saritap@vmware.com>
```

Comment:
The author listed here is not consistent with the username used in the commit message.  They should be aligned to some degree

Reply:
Done the changes accordingly